### ISSUE TITLE
add_new_book

### ISSUE THREAD

**Issue by otocampos** [flutter/website#11096]
Hello, Flutter Team!

I would like to request the inclusion of my book, "[Cracking the Flutter Interviews: More than 200 Verbal Questions and Practical Examples](https://www.amazon.com/dp/B0DFLW21K8)," in the books resources section of the official Flutter website.

Book Details:

Title: Cracking the Flutter Interviews: More than 200 Verbal Questions and Practical Examples
Author: Otniel Campos
Publication Date: August 2024
URL: https://www.amazon.com/dp/B0DFLW21K8
cover:![Otniel Campos (1)](https://github.com/user-attachments/assets/95b5fbc9-937a-41ac-8a74-9646cce74255)

Flutter version: Flutter 3.19.3
Description: This book serves as an essential tool for Flutter developers preparing for interviews. It covers a wide range of topics including data structures, architecture patterns, UI/UX, testing, and app optimization, with over 200 verbal questions and practical examples.
Motivation: The book was written to help Flutter developers prepare for interviews, covering both fundamental and advanced topics in the technology. I believe it would be a valuable resource for the Flutter community, especially for those looking to improve their skills and knowledge in this field.

**Comment by sfshaza2** [comment]
Hi, @otocampos! Congrats on your book, but it's published digitally only, right? If so, can you correct the title? Correct English is "Cracking the Flutter Coding Interview." (singular S)

HOWEVER, it would be even more clear if you called it "Cracking the Flutter Coding Interview." Could you make this change?

**Comment by otocampos** [comment]
hi @sfshaza2  , thanks for congrats .Yes it´s published digitally only. Tks for your suggestion, i really appreciated it. I will make these changes and text you

**Comment by otocampos** [comment]
@sfshaza2 could you help me again? I thought about this discussion and I'm thinking that "coding interview" won't be that clear, because in the book I'm covering verbal questions, some DSA tips and small real-world projects, plus HR tips not just coding challenges. Do you think "coding interview" still makes sense? Once again, thanks for the tips and i am open to new suggestions.

**Comment by sfshaza2** [comment]
Hmmm. Well, you _could_ change it to "Cracking the Flutter interview", which is grammatically correct. I'm sure you have some reviewers. What do they think?

**Comment by otocampos** [comment]
I had guidance from reviewers from the publisher packt, the original name was "The Ultimate Guide to Flutter Interviews", but they are no longer in this project. So I decided to change the original name.

**Comment by sfshaza2** [comment]
Yes "The Ultimate Guide to Flutter Interviews" is grammatically correct.  :)

**Comment by otocampos** [comment]
Hi @sfshaza2, thanks for supporting me, I already made the changes to my book, but now I need to wait 72 hours for Amazon to apply the changes., I changed the title and cover of the book

**Comment by otocampos** [comment]
hi , the updates are done, pls check here:
https://www.amazon.com/dp/B0DFLW21K8

**Comment by sfshaza2** [comment]
WOOT!!
